docker/ubuntu-16.04/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM  ubuntu:16.04
+LABEL maintainer "Gemfield <gemfield@civilnet.cn>"
+
+RUN apt-get update && \
+        DEBIAN_FRONTEND="noninteractive" \
+        apt-get install -y vim git cmake g++ autoconf libtool gperf pkg-config binutils tzdata && \
+        apt-get autoremove -y && \
+        apt-get clean -y
+
+ENV RESIP_DIR=/opt/resiprocate
+WORKDIR /workspace
+RUN git clone https://github.com/resiprocate/resiprocate
+RUN cd resiprocate && \
+          git submodule update --init --recursive && \
```

Comment:
did you see the build/debian.sh script?

Reply:
Yes, I will update the Dockerfile according to this script.

Reply:
@dpocock is there a apt way to install libsrtp-dev on Debian buster(seems only libsrtp2 can be found on this release...)?
Now there has a relate compile issue:
```bash
Flow.hxx:16:10: fatal error: srtp/srtp.h: No such file or directory
```